### ISSUE TITLE
Networking logs are spamming the console

### ISSUE THREAD

**Issue by GalRogozinski** [iotaledger/iri#1237]
### Bug description
Currently the logs in TCP neighbor:
```
synchronized (sendQueue) {
            if (sendQueue.remainingCapacity() == 0) {
                sendQueue.poll();
                log.info("Sendqueue full...dropped 1 tx");
            }
            log.info("Sendqueue size: {}",sendQueue.size());
            byte[] bytes = packet.getData().clone();
            sendQueue.add(ByteBuffer.wrap(bytes));
        }
```

Best solution will be to fix the `IntervalLogger` interface to that of a standard logger and then use the following pattern:
```
if (intervalLog.isDebug()) {
   intervalLog.debug(...)
}
```
Maybe use `trace` level instead.

Are spamming the console.
### IRI version
1.5.5

### Steps To Reproduce
1. Configure node A to peer to Node B but don't configure to peer to A

### Expected behaviour
Naive:Log should not be printed at all
Better: be printed intermittently in `debug` or `trace` mode.

### Actual behaviour
Console is being overflown

**Comment by gjeee** [comment]
yeah sorry this was my mistake......i tagged you a while ago (in PR#940) to fix this but i got no response. hans moog already corrected this in one of his PRs :-) but maybe better to do this in a separate PR.

**Comment by GalRogozinski** [comment]
Yup,
but this is how Hans did it. I will keep this issue open because I think using the `IntervalLogger` could still be useful
